### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by WilliamCampbellie** [cg2121/obs-advanced-timer#31]
Would it be too much to ask to get some information on how to actually install and start to use this?

**Comment by rdwulfe** [comment]
Yeah, it'd be nice for those of us getting started. This looks like a wonderful tool, no idea how to use it, and nothing in it seems to explain itself, other than formatting.
